### DISCUSSION TITLE
Table of contents changes sizes.

### DISCUSSION THREAD

**Discussion by rachfop** [thuliteio/doks#1016] [General]
The left-hand navigation (table of contents) flexes or changes sizes for no apparent reason. 

Here's a screenshot of the normal TOC.

<img width="1144" alt="custom-drivers" src="https://user-images.githubusercontent.com/64233065/221435516-80fc733e-dc32-4bfe-8ce9-0688df506fb3.png">

And the markdown:

```md
---
title: "Custom Drivers"
description: "Learn to establish connections between Humanitec and your infrastructure."
lead: "Drivers are used to establish the connection between Humanitec and your infrastructure."
date: 2023-02-06T13:07:43-08:00
lastmod: 2023-02-06T13:07:43-08:00
draft: false
type: docs
images: []
menu:
  docs:
    parent: "work-with-drivers"
    identifier: "drivers-d1f98604c29231c0ba8d8ba99dfb7e90"
weight: 405
toc: true
---
```

Here's a screen shot of the not, so normal TOC:

<img width="1082" alt="drivers" src="https://user-images.githubusercontent.com/64233065/221435584-c501aac4-9098-46e4-a6e4-5d660a20c66e.png">

And the markdown:

```md
---
title: "Resource Drivers"
description: "Learn to work with Resource Drivers."
lead: ""
date: 2023-02-06T13:07:07-08:00
lastmod: 2023-02-06T13:07:07-08:00
draft: false
type: docs
images: []
menu:
  docs:
    parent: "work-with-drivers"
    identifier: "writing-54f1351ca3e22a26d41b419347b5c048"
weight: 400
toc: true
---
```

I don't like the second page, how the TOC collapses a bit and it is indented weird.

Any suggestions on how to fix this, as there seems to be a lot of pages setup like this.

**Comment by h-enk** [comment]
I can't reproduce this issue. Do you have a repo you could share?
